.github/workflows/build-and-deploy.yml

```diff
@@ -60,6 +60,57 @@ jobs:
           message: |
             Review app deployed to ${{ steps.deploy.outputs.environment_url }}
 
+  deploy_v2_review:
+    name: Deploy to review_aks environment
+    concurrency: deploy_v2_review_${{ github.event.pull_request.number }}
+    needs: [docker]
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy_v2')
+    environment:
+      name: review_aks
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/actions/deploy_v2
+        id: deploy_v2
+        with:
+          environment: review_aks
+          docker_image: ${{ needs.docker.outputs.docker_image }}
+          azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+          arm-access-key: ${{ secrets.ARM_ACCESS_KEY }}
+          pr-id: ${{ github.event.pull_request.number }}
+
+      - name: Post sticky pull request comment
+        if: github.event_name == 'pull_request'
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          message: |
+            AKS review app deployed to ${{ steps.deploy_v2.outputs.environment_url }}
+
+  deploy_v2_dev:
+    name: Deploy to development_aks environment
+    concurrency: deploy_v2_development
+    needs: [docker]
+    runs-on: ubuntu-latest
```

Comment:
add 'continue-on-error: true' as we don't want an aks dev deploy failure to stop paas deploys